pages/national_resources/si_resources.md

```diff
@@ -55,12 +55,27 @@ national_resources:
 <!---Please focus on resources that are relevant for the whole country for life sciences--->
```

Comment:
@nadja-zlender, can you kindly fill in metadata part of the page e.g. contributors' names etc? Thanks